build.sbt

```
@@ -64,7 +64,7 @@ ThisBuild / compile / javacOptions ++= Seq(
 ThisBuild / scalacOptions ++= Seq(
   "-deprecation", // Emit warning and location for usages of deprecated APIs.
   "--release",
-  "11",
+  "11"
```

To avoid unrelated change
```suggestion
  "11",
```

---

This is formattee change. It has removed the unwanted last `,`